skyportal/onboarding.py

```diff
@@ -110,19 +120,42 @@ def setup_invited_user_permissions(strategy, uid, details, user, *args, **kwargs
     group_ids = [g.id for g in invitation.groups]
 
     existing_user_group_ids = [g.id for g in existing_user.groups]
+
+    # grab stream_ids
+    stream_ids = set()
+    for group_id in group_ids:
+        streams = (
+            DBSession()
+            .query(Stream)
+            .join(GroupStream)
+            .filter(GroupStream.group_id == group_id)
+            .all()
+        )
+        for stream in streams:
+            stream_ids.add(stream.id)
+
+    # Create single-user group
+    DBSession().add(Group(name=user.username, users=[user], single_user_group=True))
+    # Add stream access to single user group
+    single_user_group = (
```

Comment:
Minor thing that can wait til later -- we can just assign this to the group instance being added above, and won't need to re-fetch it here